.travis.yml

```diff
@@ -1,7 +1,6 @@
 before_install: gem install bundler
 language: ruby
 rvm:
-  - 2.3
```

Comment:
Our latest Rubocop rule is not compatible with Ruby 2.3 so I'm dropping support since support for Ruby 2.3 has already ended.